introduction.md

```diff
@@ -1,3 +1,6 @@
 ## A short intro
 ### Favorite Code School path
-HTML & CSS
\ No newline at end of file
+HTML & CSS
+### Skills
+* Git
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?